python/python-quiz.md

```diff
@@ -2676,3 +2676,16 @@ The code defines a `square` function to calculate the square of a number. It the
 - [ ] getline()
 
 [Reference](https://www.geeksforgeeks.org/taking-input-from-console-in-python/)
+
+
+#### Q185.1. What will be the output of the following Python code?
+
+print("Hello {name1} and {name2}".format(name1='foo', name2='bin'))
```

Comment:
write in code format

`print("Hello {name1} and {name2}".format(name1='foo', name2='bin'))`

Reply:
it is in code format only